packages/store-ui/package.json

```diff
@@ -43,15 +43,15 @@
     "@vtex-components/accordion": "^0.2.3",
     "@vtex-components/drawer": "^0.2.4",
     "deepmerge": "^4.2.2",
-    "gatsby-link": "^3.5.0",
     "react-hook-inview": "^4.3.9",
-    "react-intl": "^5.17.7",
+    "react-intl": "^5.18.0",
     "react-swipeable": "^6.0.0",
     "reakit": "^1.3.0",
     "text-mask-core": "^5.1.2",
     "theme-ui": "^0.3.1"
   },
   "peerDependencies": {
+    "gatsby-link": "^3.5.0",
```

Comment:
why? 😭

Reply:
making it a peer-dep, yarn won't try to hoist this and we will have only one single instance of gatsby-link in the final node_modules, removing the duplicated package

Reply:
I want to be able to remove this dependence one day